vault/identity_store_entities.go

```diff
@@ -579,7 +579,7 @@ func (i *IdentityStore) handleEntityDeleteCommon(ctx context.Context, txn *memdb
 	// internal and external
 	groups, err := i.MemDBGroupsByMemberEntityIDInTxn(txn, entity.ID, true, false)
 	if err != nil {
-		return nil
+		return err
```

Comment:
I'm not sure why this previously returned `nil`

Reply:
It looks like a typo mistake. Thanks for catching it!